packages/fluentui/react-bindings/test/hooks/useCSS-test.tsx

```diff
@@ -0,0 +1,143 @@
+import { useCSS, UseCSSStyleInput, Unstable_FluentContextProvider } from '@fluentui/react-bindings';
+import { mount } from 'enzyme';
+// @ts-ignore No typings :(
+import * as prettier from 'prettier';
+import * as React from 'react';
+import { RendererRenderGlobal } from '@fluentui/react-northstar-styles-renderer';
+
+expect.addSnapshotSerializer({
+  test(value) {
+    return value?._isMockFunction;
+  },
+  print(value: jest.Mock) {
+    const css = value.mock.calls.map(call => call[0]).join(' ');
+
+    return prettier.format(css, { parser: 'css' }).trim();
+  },
+});
+
+const TestComponent: React.FC<{ styles: UseCSSStyleInput[] }> = props => {
+  const className = useCSS(...props.styles);
+
+  return <div className={className} />;
+};
+
+function getMountOptions(renderGlobal: RendererRenderGlobal, rtl: boolean = false) {
+  return {
+    wrappingComponentProps: {
+      value: {
+        renderer: { renderGlobal },
+        rtl,
+      },
+    },
+    wrappingComponent: Unstable_FluentContextProvider,
+  };
+}
+
+describe('useCSS', () => {
+  it('returns a className that contains', () => {
+    const styles = [{ color: 'red' }];
+    const wrapper = mount(<TestComponent styles={styles} />);
+
+    expect(wrapper.find('div').prop('className')).toBeDefined();
```

Comment:
On the other hand shouldn't we also test what that `className` is, or what styles the component has.

Reply:
Changed it as `className` is stable